### PR TITLE
Add install instructions for the workspace catalog repo

### PR DESCRIPTION
**Pull request by cbuto**
## Description of changes being made

Adding docs about how to install the workspace catalog since this will be a manual task for now until https://jira.d2iq.com/browse/D2IQ-79950 is resolved. I just don't want this to get lost!

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Create new PRs against `develop`, or `main` for hotfixes to production.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

File: pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/index.md [hunk 1]
```diff
@@ -8,6 +8,38 @@ excerpt: Workspace Catalog Applications
...
+
+1.  Create the `GitRepository`:
+
+    ```sh
+    kubectl apply -f - <<EOF
+    apiVersion: source.toolkit.fluxcd.io/v1beta1
+    kind: GitRepository
+    metadata:
+      name: dkp-catalog-applications
+      namespace: ${WORKSPACE_NAMESPACE}
```
Comment (cwyl02) [inline pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/index.md:27]: include the catalog label:

**Review by cwyl02 — CHANGES_REQUESTED** [review]
include the catalog label under `metadata`:
```yaml
labels: 
    kommander.d2iq.io/gitapps-gitrepository-type: catalog
```

**Comment by cbuto** [comment]
> include the catalog label under `metadata`:
> 
> ```yaml
> labels: 
>     kommander.d2iq.io/gitapps-gitrepository-type: catalog
> ```

Good call! added in 9e38e22cbfee59a120b76c87f096954791eb0fac

**Comment by cbuto** [comment]
moved to https://github.com/mesosphere/dcos-docs-site/pull/3973

**Pull request closed without merging**